Calculator/Calculator.cpp

```diff
@@ -226,12 +331,9 @@ int main() {
 }
 
 /*
-Notes:  Need help figuring out string formatting so character variable can be inserted in error messages.
-
-        When implementing () handling, negatives relating to () will hang in equation
-        Implement function to call from calculator functions to process negatives - 
-        if operator is followed by negative, call process_negative
-        process_negative replaces negative and number entries with a negative number entry
+Notes:  Currently empty parentheses - () - are replaced with a zero.
+        What is the proper way to handle them? Should an error be thrown? Or is that acceptable?

```

Comment:
empty parens should evaluate to the same thing an empty statement evaluates to, if that's zero, that's fine. if it's an error, that's also fine.